server/app/middlewares/websocket_backend.rb

```diff
@@ -172,30 +173,28 @@ def find_node(req)
   # @param [Faye::WebSocket] ws
   # @param [Rack::Request] req
   def on_open(ws, req)
+    node_id = req.env['HTTP_KONTENA_NODE_ID']
+    agent_version = req.env['HTTP_KONTENA_VERSION'].to_s
+    connected_at = req.env['HTTP_KONTENA_CONNECTED_AT']
+    connected_at = Time.parse(connected_at) if connected_at
```

Comment:
Wouldn't this leave `connected_at` as `nil` if the `HTTP_KONTENA_CONNECTED_AT` is not set in request? And having it nil would probably cause some side effects. :) Is it possible that this is not set in the request?